src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.cs

```diff
@@ -192,35 +204,41 @@ private static void CreateCSP(CspParameters parameters, bool randomKeyContainer,
         /// </summary>
         public static bool CryptAcquireContext(out SafeProvHandle psafeProvHandle, string pszContainer, string pszProvider, int dwProvType, uint dwFlags)
         {
-            return Interop.CryptAcquireContext(out psafeProvHandle, pszContainer, pszProvider, dwProvType, dwFlags);
+            bool ret = Interop.CryptAcquireContext(out psafeProvHandle, pszContainer, pszProvider, dwProvType, dwFlags);
+            return ret;
```

Comment:
Oops.  There used to be a registration line between them.  (Tracking object creation for the callstack of where things were getting finalized instead of disposed).  I'll revert this.